src/com/theironyard/novauc/Car.java

```diff
@@ -0,0 +1,43 @@
+package com.theironyard.novauc;
+
+import java.util.Currency;
+
+/**
+ * Created by peter on 2/7/17.
+ */
+public class Car {
+    double cost;
+    double horsePower;
+    String color;
+    char model;
+    boolean fourDoors;
+
```

Comment:
Nice job Peter.  Everything looks good.  In the future pay a bit more attention to the requirements.  In each class, write setters and getters for ALL of its fields.  You have demonstrated that you understand how to do this. So no need to resubmit.